boards/arm/96b_carbon_nrf51/96b_carbon_nrf51.dts

```diff
@@ -23,3 +23,11 @@
 	current-speed = <115200>;
 	status = "ok";
 };
+
```

Comment:
curious why you enabled any i2c on this board, nothing in the existing code is enabling i2c by default for the board